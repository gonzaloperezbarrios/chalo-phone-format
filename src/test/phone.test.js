import React from "react";
import Phone from "../index";
import renderer from "react-test-renderer";

test("Phone basic snapshot", () => {
  const component = renderer.create(
   <Phone
      value="2151234455"
      handleChange={console.log}
      isFormatOK={true}
      format="(###) ###-####"
      mask="-"
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
