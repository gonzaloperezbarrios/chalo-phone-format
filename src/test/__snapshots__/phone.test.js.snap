// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Phone basic snapshot 1`] = `
<input
  onBlur={[Function]}
  onChange={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onMouseUp={[Function]}
  type="text"
  value="(215) 123-4455"
/>
`;
